new-pipeline/src/dist.js

```diff
@@ -869,15 +869,22 @@ function displayHistograms(histogramsList, dates, useTable, cumulative, trim) {
     var channel = $("#channel-version").val().split("/")[0]
     var desc = getDescription(metric, channel, description);
     var link = getDescriptionLink(metric, channel, description);
-    if (metric != desc) {
```

Comment:
Removing this changes the existing behavior.
I think we should just add all the use counter link handling outside of this  `if (metric != desc) ... else ...` , after the `$('#dist-caption-link').html(link);`.